src/protos/recipe_search_service.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package recipe;
+
+service RecipeSearchService {
+  rpc SearchRecipes (SearchRequest) returns (SearchResponse);
+}
+
+message SearchRequest {
+  string query_type = 1;
```

Comment:
we should use Enum in this proto file to validate query_type instead of doing that in elasticsearch_client.py:
```
message SearchRequest {
  enum QueryType {
    MATCH = 0;
    MULTI_MATCH = 1;
    MATCH_PHRASE = 2;
    TERM = 3;
    EXISTS = 4;
    RANGE = 5;
  }
  QueryType query_type = 1;
  ```